specification/apicenter/resource-manager/Microsoft.ApiCenter/preview/2024-06-01-preview/apicenter.json

```diff
@@ -4464,7 +4608,7 @@
         "msiResourceId": {
```

Comment:
In _"/importApiSource"_ API, `AzureApiManagementSource` is different from that in _"/apiSources"_. Which means that line 4603 should be **`resourceId`** in _"/apiSources"_ while it should be **`sourceResourceIds`**, the only key name the server side acceptable, in _"/importApiSource"_.